.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
       - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: nightly
```

Comment:
Latest nightly refuses to compile the code. This is a temporary workaround.
![image](https://github.com/user-attachments/assets/5ea848b2-3528-4c0a-8f0a-88d6ece36996)